eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/CDDL.hs

```diff
@@ -225,14 +213,14 @@ dns_name = "dns_name" =:= VText `sized` (0 :: Word64, 64 :: Word64)
 
 single_host_addr :: Named Group
 single_host_addr =
-  "single_host_addr" =:~ grp [0, port / VNil, ipv4 / VNil, ipv6 / VNil]
+  "single_host_addr" =:~ grp [0, a $ port / VNil, a $ ipv4 / VNil, a $ ipv6 / VNil]
```

Comment:
What does the `a` do ?

Reply:
It turns any type that implements `IsType0` into a group/array entry.
```haskell
a :: (IsType0 a, IsGroupOrArrayEntry e) => a -> e
```